examples/xlnet/xlnet_classification_main.py

```diff
@@ -110,22 +123,34 @@ def evaluate(model, iterator, is_regression: bool = False, print_fn=None,
 def construct_datasets(args, device: Optional[torch.device] = None) \
         -> Dict[str, tx.data.RecordData]:
     sp_model = spm.SentencePieceProcessor()
-    sp_model.Load(args.spm_model_path)
+
+    pretrained_model_dir = tx.modules.load_pretrained_xlnet(
+        pretrained_model_name=args.pretrained_model_name,
+        cache_dir='xlnet_pretrained_models')
+
+    spm_model_path = os.path.join(pretrained_model_dir, "spiece.model")
+    sp_model.Load(spm_model_path)
 
     cache_prefix = f"length{args.max_seq_len}"
-    tokenize_fn = xlnet.data.create_tokenize_fn(sp_model, args.uncased)
-    processor_class = xlnet.data.get_processor_class(args.task)
+    tokenize_fn = data_utils.create_tokenize_fn(sp_model, args.uncased)
+    processor_class = processor.get_processor_class(args.task)
     data_dir = args.data_dir or f"data/{processor_class.task_name}"
     cache_dir = args.cache_dir or f"processed_data/{processor_class.task_name}"
     task_processor = processor_class(data_dir)
-    xlnet.data.construct_dataset(
-        task_processor, cache_dir, args.max_seq_len,
-        tokenize_fn, file_prefix=cache_prefix)
-
-    datasets = xlnet.data.load_datasets(
-        args.task, cache_dir, args.max_seq_len, args.batch_size,
-        file_prefix=cache_prefix, eval_batch_size=args.eval_batch_size,
-        shuffle_buffer=None, device=device)
+    dataset.construct_dataset(task_processor,
+                              cache_dir,
+                              args.max_seq_len,
+                              tokenize_fn,
+                              file_prefix=cache_prefix)
+
+    datasets = dataset.load_datasets(args.task,
```

Comment:
Can we reformat this part of code to the previous style, i.e. wrap before the first argument? This would result in way fewer lines and make the code look less "sparse".